gobject-subclass/src/object.rs

```diff
@@ -60,7 +60,7 @@ macro_rules! box_object_impl(
     };
 );
 
-pub trait ImplTypeStatic<T: ObjectType>: Send + Sync + 'static {
```

Comment:
This will also be required for non-Send/Sync types (just not for GTK). E.g. you can create a GIO stream from any thread you want (and this would then use the `new` function below), but the returned stream is neither Send nor Sync.